### PR TITLE
fix: #tno-1459 - channel was not being updated

### PR DESCRIPTION
**Pull request by kylgarmor**
- switch to using react-router-dom, inline with the content page
- also refactored loading state

### CODE REVIEW

File: app/editor/src/features/content/papers/Papers.tsx [hunk 1]
```diff
@@ -120,21 +117,26 @@ export const Papers: React.FC<IPapersProps> = (props) => {
...
     async (filter: IPaperFilter & Partial<IContentListAdvancedFilter>) => {
       try {
-        setLoading(true);
-        const data = await findContent(
-          makeFilter({
-            ...filter,
-          }),
-        );
-        return new Page(data.page - 1, data.quantity, data?.items, data.total);
+        if (!isLoading) {
```
Comment (CelloLiu) [inline app/editor/src/features/content/papers/Papers.tsx:120]: If isLoading == true, the finally section will set it to false right away, is that what expected?

File: app/editor/package.json [hunk 2]
```diff
@@ -122,7 +122,8 @@
     ],
     "rules": {
       "simple-import-sort/imports": "error",
-      "simple-import-sort/exports": "error"
+      "simple-import-sort/exports": "error",
+      "no-console": ["error", { "allow": ["error", "debug"] }]
```
Comment (Fosol) [inline app/editor/package.json:126]: Thanks

File: app/editor/src/features/content/papers/Papers.tsx [hunk 3]
```diff
@@ -120,21 +117,24 @@ export const Papers: React.FC<IPapersProps> = (props) => {
...
       } catch (error) {
         // TODO: Handle error
         throw error;
       } finally {
-        setLoading(false);
+        setIsLoading(false);
       }
     },
-    [findContent],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Fosol) [inline app/editor/src/features/content/papers/Papers.tsx:136]: What is being ignored in the dependencies?  We always add documentation to ensure we're not introducing a hidden bug.

File: app/editor/package.json (hunk 2 above)
Comment (jdtoombs) [reply app/editor/package.json:126]: Gonna be handy :)

**Pull request merged**